src/main/java/com/github/dockerjava/api/model/AuthConfig.java

```diff
@@ -94,39 +94,26 @@ public int hashCode() {
 
     @Override
     public boolean equals(Object obj) {
-        if (this == obj)
-            return true;
-        if (obj == null)
-            return false;
-        if (getClass() != obj.getClass())
-            return false;
+        if (this == obj) return true;
+        if (obj == null) return false;
+        if (getClass() != obj.getClass()) return false;
         AuthConfig other = (AuthConfig) obj;
         if (auth == null) {
-            if (other.auth != null)
-                return false;
-        } else if (!auth.equals(other.auth))
-            return false;
+            if (other.auth != null) return false;
+        } else if (!auth.equals(other.auth)) return false;
         if (email == null) {
-            if (other.email != null)
-                return false;
-        } else if (!email.equals(other.email))
-            return false;
+            if (other.email != null) return false;
+        } else if (!email.equals(other.email)) return false;
         if (password == null) {
-            if (other.password != null)
-                return false;
-        } else if (!password.equals(other.password))
-            return false;
+            if (other.password != null) return false;
+        } else if (!password.equals(other.password)) return false;
         if (serverAddress == null) {
-            if (other.serverAddress != null)
-                return false;
-        } else if (!serverAddress.equals(other.serverAddress))
-            return false;
+            if (other.serverAddress != null) return false;
+        } else if (!serverAddress.equals(other.serverAddress)) return false;
         if (username == null) {
-            if (other.username != null)
-                return false;
-        } else if (!username.equals(other.username))
-            return false;
+            if (other.username != null) return false;
+        } else if (!username.equals(other.username)) return false;
         return true;
     }
-    //CHECKSTYLE:ON
+    // CHECKSTYLE:ON
```

Comment:
no neede in formatting equals and hashcode that wouldn't never match to auto-generated and there is no sense in manual cleanups.